arm9/source/patches/platform/datel/DATELReadSectorsAsm.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+#include "sections.h"
+#include "../SdReadPatchCode.h"
+DEFINE_SECTION_SYMBOLS(datel_read);
+
+extern u16 DATEL_readSectorSdhcLabel;
+
+extern u32 DATEL_SDReadMultipleSector_SpiSendSDIOCommandR0;
+
+extern u32 DATEL_SDReadMultipleSector_ReadSpiByteTimeout;
+extern u32 DATEL_SDReadMultipleSector_ReadSpiByte;
+
+extern "C" void DATEL_SDReadMultipleSector(u32 srcSector, void* dst, u32 sectorCount);
+
+class DATELReadSdPatchCode : public SdReadPatchCode
```

Comment:
```suggestion
class DatelReadSdPatchCode : public SdReadPatchCode
```
Also rename the file and the other files such that it's Datel, rather than DATEL (you can use git mv for these kind of case changes)